config/database.yml.dist

```diff
@@ -1,27 +0,0 @@
-# Warning: The database defined as "test" will be erased and
-# re-generated from your development database when you run "rake".
-# Do not set this db to the same as development or production.
-test:
-  adapter: sqlite3
-  database: db/test.sqlite3
-  pool: 5
-  timeout: 5000
-
-development:
-  adapter: postgresql
-  database: bfr_webapp_db
-  username: bfr_webapp_user
-  password: CHANGEME
-  host: localhost
-  pool: 5
-  timeout: 5000
-
-production:
-  adapter: postgresql
-  database: bfr_webapp_db
-  username: bfr_webapp_user
-  password: CHANGEME
-  host: localhost
-  pool: 5
-  timeout: 5000
-
```

Comment:
@ericfreese Thank you for the comment. You are right. I just realized that `database.yml` was listed in `.gitignore`. I will have this corrected.